src/core/inc/config.h

```diff
@@ -90,6 +90,7 @@ struct vm_config {
     struct {
         uint64_t budget;
         uint64_t period_us;
+        uint64_t ticket_num;
```

Comment:
```suggestion
        uint64_t num_ticket;
```
IMO, we should start to follow a name convention. As this represents the number of tickets available for the VM, maybe it would make sense to call it num_tickets

Reply:
I'll change all of this to follow the approach then.